src/components/Switch/index.js

```diff
@@ -37,7 +38,7 @@ export function useHiddenSwitch({
     isChecked: overrideDefaultChecked || checked,
     HiddenSwitch: (
       <input
-        ref={ref}
+        ref={inputRef}
```

Comment:
this might be breaking change, please use mergedRef

Reply:
or was it a bug?

Reply:
It was a bug. `ref` was used, though it was not defined